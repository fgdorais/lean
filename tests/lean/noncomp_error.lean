noncomputable definition a := 2
