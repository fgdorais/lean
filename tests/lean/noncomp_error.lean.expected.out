noncomp_error.lean:1:0: error: definition 'a' was incorrectly marked as noncomputable
